### ISSUE TITLE
Cannot parse $VERSION from... in newly minted module

### ISSUE THREAD

**Issue by gimme00** [Leont/App-ModuleBuildTiny#45]
```bash
mbtiny mint Test::MBTinyTest --dirname=test-mbtinytest
```

results in

```bash
Cannot parse $VERSION from lib/test/mbtinytest.pm at <snip>.perlbrew/libs/perl-5.30.3@doseme/lib/perl5/App/ModuleBuildTiny/Dist.pm line 238.
```

Trying again: 

```bash
cd test-mbtinytest
mbtiny regenerate
```

Again, you get:

```bash
Cannot parse $VERSION from lib/test/mbtinytest.pm at <snip>.perlbrew/libs/perl-5.30.3@doseme/lib/perl5/App/ModuleBuildTiny/Dist.pm line 238
```

```bash
cat lib/test/mbtinytest.pm
```

Version is there. Error persists even if set to 0.001, which is supposed to be the default.

```perl
package Test::MBTinyTest;

use strict;
use warnings;

our $VERSION = '0.000';

1;

__END__

=pod
<snip>
```

**Comment by Leont** [comment]
Right. It takes the dirname for the directory name unless specified in the metamerge file. I guess it should write such a metamerge file when using an alternative dirname.

**Comment by Leont** [comment]
This was fixed in 0.037
